pangeo_forge_recipes/recipes/reference_hdf_zarr.py

```diff
@@ -149,15 +153,18 @@ class HDFReferenceRecipe(BaseRecipe, StorageMixin, FilePatternMixin):
     #  store_chunk, finalize_target. The strategy was used for NWM's 370k files.
 
     # TODO: as written, this recipe is specific to HDF5 files. kerchunk
-    #  also supports TIFF and grib2 (and more coming)
+    #  also supports TIFF, FITS, netCDF3 and grib2 (and more coming)
```

Comment:
Could we use the new `file_type` attribute for validation here? See #322

Reply:
Sure; but of course it would be nice to extend this recipe type to cope with all the possible kerchunk scanners.